libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -69,12 +68,20 @@ export class ChatsApi extends BaseAPI {
     if (requestParameters?.cursor) {
       queryParameters.offset = requestParameters.cursor
     }
-    return (await this.request({
+    const response = (await this.request({
       method: 'GET',
       path,
       headers: await this.getSignatureHeader(path),
       query: queryParameters
     })) as TypedCommsResponse<UserChat[]>
+
+    const unencrypted = await Promise.all(
+      response.data.map(async (c) => await this.decryptLastChatMessage(c))
```

Comment:
nit: i could be wrong but you probably don't need the await inside the map since you are awaiting the promise.all. it may even _maybe_ be faster without it